### ISSUE TITLE
UST From ETH to SOL Stuck at Awaiting Completion on Solana

### ISSUE THREAD

**Issue by amol3335** [solana-labs/oyster#513]
Hi, Please resolve this issue for transaction
https://etherscan.io/tx/0x90e457df6271463a9f499b6c28052fdd3f65b34e6a4eb27d32acf6b06a88603d

**Comment by amol3335** [comment]
Hi @leoluk Sir What's the issue Can You Please tell me? I am so Unlucky I get Stuck Everytime. I literally Can't even Eat Food When Such thing happens. I know you will sort this out and I will never ever again Gonna Use V1 again.

**Comment by bagz11** [comment]
@leoluk  

I know you can help this guy with this problem. Please help!

**Comment by amol3335** [comment]
Thanks @bagz11 I don't even know You. But @leoluk and Other devs at Wormhole Don't feel for users. Because it's their Will. Since Last 10 days My mental health has completely devastated. I have thought of Committing Suicide Every Day and this is know by their Discord Admin Susu. I have so many Problems in My Life. My Mother Committed Suicide 12 years Back. Now, My Brother at the age of 32 Has Diabetes and He recently Had a TIA attack. I don't have a Job or anything else to do. 
Many Users are affected by this. One of user got liquidated on borrow position. @leoluk If You guys had knew that this is gonna take so many days then You guys should have provided funds to users like you compensated 120K ETH.
@hendrikhofstadt I really had hopes from You. But Nowadays Everyone is so Busy that they don't have time for Others. If I take any step of Committing Suicide Wormhole Team will Be the Only responsible for it.

**Comment by amol3335** [comment]
How Can be a team so Heartless that In last 10 days I got fearful of asking Help. The Most Disappointing thing was Since day 1 Discord admins were saying it won't take more than week. Now It's almost 2 weeks.  I was really afraid of asking Help to @hendrikhofstadt @leoluk and Thanks to You guys Because You made me mentally sick. I am Almost mad now and I can End My Life Anytime now.

**Comment by bagz11** [comment]
amol3335 , we don't know each other, but I understand how you feel. I was in the same situation, all my money (which I earned 2 years) was stuck in this bridge (V1) and there was nothing I could do. I couldn't sleep, eat. I have been trying to reach the admins for several days.

I am sure that your situation will be resolved (because such situations have been resolved). gotta get some attention devs @leoluk @yamijuan @bartosz-lipinski

**Comment by leoluk** [comment]
Yes - the funds are safe and our team is working on completing the transactions. We definitely feel for you and it sucks to have funds in limbo! Safely replacing the funds would take longer than getting them unstuck. I'm just one engineer of many working on this project, but rest assured that everyone is working hard to clear out the backlog on v1 and v2.

The reason it's taking so long is we had to add new functionality to the core node software first. We can't just push a button, we first had to add the button. This takes time - it needs testing, review and then external audits, since it involves changes to a critical path in the code base. The audits were completed on Friday and we're now backporting it to v1 and rolling it out.

**Comment by amol3335** [comment]
Honestly, We Should have got this response way earlier. But In My Case I used the Bridge after the announcement that Everything has sorted out. And after that We get to hear from Admins it's gonna take days. Earlier they said few days, then they said Wednesday Thursday, then they said End of the week and now it's early next week. Also I felt being Discriminated because Everytime I have been asked to raise github ticket but there are some trxns for which there's no ticket raised. 
I am a Guys Who is trying very hard for the Family and Don't Do any other work apart from trading. 
Doing Trading is like Food to Me. So, How I am gonna live days without it. Also Thank God this wasn't My Borrowed Funds If That was the Case I definitely wouldn't have been typing here.
Also As a Request You guys Should Compensate Users For this.

**Comment by amol3335** [comment]
@leoluk Lost All the Hopes Today. You Guys Replenished 120K ETH in 1 Day but Could help For 1.2Mn in 12 Days. I am going to take the Last step to End my life. Thanks to Wormhole Team for making me Do it.

**Comment by joelyer** [comment]
@bagz11 i saw that you had a similar issue to me before. Was it resolved?

@leoluk will the new invention get coins stuck in v1 wormhole unstuck?

**Comment by leoluk** [comment]
https://explorer.solana.com/tx/L9zz5khbdhGBRz5PPDxYMRWoBtD2nMfyn8uDJau6V8Ajqen9x8TrGN2cz8bzwoeFFhYnHwu7oNpgobAHU1QFwcZ

> @leoluk will the new invention get coins stuck in v1 wormhole unstuck?

Yes!
